packages/core/src/websockets/constants.ts

```diff
@@ -3,7 +3,7 @@
 // All this can be seen in the browser. If you want something private
 // put in other place.
 
-import type { Message } from '@latitude-data/constants/legacyCompiler'
+import type { Message } from '@latitude-data/compiler'
```

Comment:
Nope, escape the types if it gives you incompatibilities but don't import from compiler